power_grid_model_c/power_grid_model_c/include/power_grid_model_c/meta_data.h

```diff
@@ -160,15 +160,10 @@ PGM_API char const* PGM_meta_attribute_name(PGM_Handle* handle, PGM_MetaAttribut
  *
  * @param handle
  * @param attribute pointer to attribute
- * @return  Type of the attribute in char const*. The string is a valid C type name. The pointer is permanantly valid.
+ * @return  Type of the attribute as in enum PGM_CType.
  *
- * Valid types are:
- *   - int32_t
- *   - int8_t
- *   - double
- *   - double[3]
  */
-PGM_API char const* PGM_meta_attribute_ctype(PGM_Handle* handle, PGM_MetaAttribute const* attribute);
+PGM_API PGM_Idx PGM_meta_attribute_ctype(PGM_Handle* handle, PGM_MetaAttribute const* attribute);
```

Comment:
I know we don't have any C API users yet, but are you sure you want to break the API by removing the old function? I actually would argue providing a `str` representation as well has its advantages

Reply:
I think this is our good chance to make breaking changes, given the timing. When we have version 1.5 released and also concrete users on the C-API, we will need to be more careful about breaking changes.